app/models/talentlms/facade.rb

```diff
@@ -33,7 +32,7 @@ def login(user)
     # @return [String] URL to redirect the user to to login
     def create_account(user)
       login = "#{ENV['RAILS_ENV']}_#{user.id + Integer(ENV.fetch('DEV_OFFSET', 0))}"
-      password = SecureRandom.hex(8)
+      password = random_password(16)
```

Comment:
with the default being 16 below, we can probably just do:
```suggestion
      password = random_password
```

Reply:
this has been adjusted